src/i18n/locales/tr.json

```diff
@@ -6,6 +6,23 @@
   "aysDescription": "Afet Yönetim Sistemi | Disaster Management System",
   "users": "Kullanıcılar",
   "home": "Ana Sayfa",
+  "page": "Sayfa",
+  "admin": {
```

Comment:
Bu şekilde obje içinde obje olarak kullanım sonradan karışıklığa yol açıyor. admin objesinden çıkarabiliriz. Genel olarak statüler her yerde kullanılabilir. Tekrar tekrar tanımlamaya gerek olmaması ve kodların kısa olması için iç içe objelerden sakınalım.